shared/chat/conversation/header/index.desktop.js

```diff
@@ -1,11 +1,11 @@
 // @flow
 import React from 'react'
-import {Box, Icon, Usernames} from '../../common-adapters'
-import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Icon, Usernames} from '../../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../../styles'
 
-import type {Props} from './header'
+import type {Props} from '.'
 
-const Header = ({muted, onOpenFolder, onShowProfile, onToggleSidePanel, sidePanelOpen, users}: Props) => (
+const ConversationHeader = ({muted, onOpenFolder, onShowProfile, onToggleSidePanel, sidePanelOpen, users}: Props) => (
```

Comment:
renamed this as the banner has a Header also which makes understanding the KBPerf() harder